components/Project/ProjectForm/index.js

```diff
@@ -377,14 +375,14 @@ export default function ProjectForm({
                 </SubTitle>
                 <ToggleYarnButton
                   left="20rem"
-                  top={`${42 + index * 11.2}rem`}
+                  top={`${55 + index * 11.2}rem`}
                   onClick={handleAddExistedYarnClick}
                 >
                   +
                 </ToggleYarnButton>
                 <ToggleYarnButton
                   left="17rem"
-                  top={`${42 + index * 11.2}rem`}
+                  top={`${55 + index * 11.2}rem`}
                   onClick={() => handleDeleteExistedYarn(yarn.id)}
```

Comment:
For easier refactoring you could save all numbers (55, 11.2) in variables and insert the variables where you use the numbers. This way you don't have to change every element but only the variable.